docs/core/testing/unit-testing-with-mstest.md

```diff
@@ -0,0 +1,215 @@
+---
+title: Use MSTest with .NET Core
+description: How to use MSTest with .NET Core
+keywords: MSTest, .NET, .NET Core
+author: ncarandini
+ms.author: wiwagn
+ms.date: 02/09/2017
+ms.topic: article
+ms.prod: .net-core
+ms.devlang: dotnet
+ms.assetid: ed447641-3e85-4e50-b7ed-004630048a3e
+---
+
+# Unit testing with MSTest and .NET Core
```

Comment:
usually our guidelines recommend on not using ing for the verbs on titles and filenames. I know we haven't followed this pattern for the pair topics so it's up to you whether you want to follow the guideline here or not.

Also this topic is having some issues with TOC. TOC link takes me to the topic but it doesn't stick once you go to the topic.